packages/ses/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "Apache-2.0",
   "type": "module",
   "main": "./dist/ses.cjs",
+  "module": "./src/main.js",
```

Comment:
Is this a rollup specific field or does it have other cataloged uses?

Reply:
Several other ESM-aware bundlers use it.

BTW, ses-integration-test was horribly broken (i.e. it didn't bundle any tests).  I at least got it working and passing again in this PR.

Reply:
Awesome. I’m hopeful that that this helps me get #220 over the line. Thank you!